.github/workflows/deploy-stubs-build-and-promote.yml

```diff
@@ -0,0 +1,27 @@
+name: Deploy stubs build and promote
+
+permissions:
+  id-token: write
+  contents: read
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - ready_for_review
+      - synchronize
+  merge_group:
+
+jobs:
+  run-checks:
+    uses: ./.github/workflows/run-checks.yml
+    secrets:
+      SAM_APP_VALIDATE_ROLE_ARN: ${{ secrets.ORCH_STUBS_SAM_APP_VALIDATE_ROLE_ARN }}
+  deploy:
+    uses: ./.github/workflows/deploy-stubs.yml
+    needs: run-checks
+    secrets:
+      GH_ACTIONS_ROLE_ARN: ${{ secrets.ORCH_STUBS_GH_ACTIONS_ROLE_ARN }}
+      ARTIFACT_BUCKET_NAME: ${{ secrets.ORCH_STUBS_ARTIFACT_BUCKET_NAME }}
+      SIGNING_PROFILE_NAME: ${{ secrets.ORCH_STUBS_SIGNING_PROFILE_NAME }}
```

Comment:
This is something I could have set better. But could we have some better naming structure for these secrets. Will need to delete the old ones and recreate for both `deploy-stubs-build-and-promote` and `deploy-stubs-dev` .

Was thinking of just having `GH_ACTIONS_ROLE_ARN` for build and promote and `DEV_GH_ACTIONS_ROLE_ARN` for dev. Since we don't need to specify orch stubs in this repo.

However, thinking in the future we will change this naming structure again as we plan to have a pipeline per stub. So will need to recreate these secrets with a structure like: `IPV_DEV_GH_ACTIONS_ROLE_ARN` , `IPV_GH_ACTIONS_ROLE_ARN`

Reply:
Sure, I think it makes sense to revisit this once we've decided what we're doing with multiple pipelines.

Reply:
To do in ticket: https://govukverify.atlassian.net/browse/ATO-938